plugins/fastboot/fu-fastboot-device.c

```diff
@@ -283,6 +284,14 @@ fu_fastboot_device_flash (FuDevice *device,
 	if (!fu_fastboot_device_read (device, NULL,
 				      FU_FASTBOOT_DEVICE_READ_FLAG_STATUS_POLL, error))
 		return FALSE;
+
+	/* actual partition flashing after download */
+	if (!fu_fastboot_device_writestr (device, flash_cmd, error))
+		return FALSE;
+	if (!fu_fastboot_device_read (device, NULL,
+				      FU_FASTBOOT_DEVICE_READ_FLAG_STATUS_POLL, error))
+		return FALSE;
+
```

Comment:
@hughsie Just as a general observation (and not that I think it should block this PR) it seems that these functions (and some others in `*_device_flash`) are using effectively what's in `fu_fastboot_device_cmd` but with a polling flag.

is it worth adding an argument that causes that behavior to `fu_fastboot_device_cmd`?

Reply:
I've done this as a follow up commit -- comments welcome.